build_debian.sh

```diff
@@ -220,6 +220,7 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     unzip                   \
     gdisk                   \
     sysfsutils              \
+    squashfs-tools          \
```

Comment:
who uses this tool?

Reply:
Apstra - we had this discussion already :-)